runtime/src/lib.rs

```diff
@@ -1979,6 +1994,7 @@ construct_runtime!(
         Content: content::{Pallet, Call, Storage, Event<T>, Config<T>},
         Storage: storage::{Pallet, Call, Storage, Event<T>, Config<T>},
         ProjectToken: project_token::{Pallet, Call, Storage, Event<T>, Config<T>},
+        ArgoBridge: argo_bridge::{Pallet, Call, Storage, Event<T>, Config<T>},
```

Comment:
Let's try placing the ArgoBridge pallet last, after Proxy. This way we should not change existing modules indexes and won't need to bump `transaction_version` (only `spec_version`)